packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -16,6 +18,10 @@ import { obfuscateNumber } from '../sms/base'
 
 const ATTESTATION_ERROR = 'Valid attestation could not be provided'
 const NO_INCOMPLETE_ATTESTATION_FOUND_ERROR = 'No incomplete attestation found'
+export const INVALID_SIGNATURE_ERROR = 'Signature is invalid'
+
+const odisPubKey = OdisUtils.Query.getServiceContext(fetchEnv('NETWORK')).odisPubKey
```

Comment:
nit: `fetchEnvOrDefault` instead of `fetchEnv` (so mainnet validators don't need to upgrade their config files, and `fetchEnv` throws an error if the env var isn't defined in the config)